cmd/virtual-kubelet/internal/commands/root/root.go

```diff
@@ -165,6 +155,17 @@ func runRootCommand(ctx context.Context, s *provider.Store, c Opts) error {
 			log.G(ctx).Debug("created new node")
 			return nil
 		}),
+	}
+	if c.EnableNodeLease {
```

Comment:
This should be the only behavior and node status updates only happen when there's something to update.